devdocs/settingUp.md

```diff
@@ -253,6 +253,7 @@ Troubleshooting instructions are given [in this document](https://docs.google.co
 * **Java Servlets**: Java Servlet technology provides a simple, consistent mechanism for extending the functionality of a Web server and for accessing existing business systems. A servlet can almost be thought of as an applet that runs on the server side--without a face.
 * **Java Data Objects (JDO)** [version 2.3; while GAE supports JDO 3.0 as well, we continue to use JDO 2.3 because it is easier to set up.]
   JDO is a standard interface for storing objects containing data into a database. The standard defines interfaces for annotating Java objects, retrieving objects with queries, and interacting with a database using transactions. An application that uses the JDO interface can work with different kinds of databases without using any database-specific code, including relational databases, hierarchical databases, and object databases.
+* **Java Persistence API (JPA)** : It  is a standard interface for accessing databases in Java, providing an automatic mapping between Java classes and database tables. 
```

Comment:
Likewise here. Try to make it similar as JDO and Datanucleus ("JPA is a ..."). The version used is v1.0, which goes with Datanucleus v1.x. Also, since Datanucleus is mentioned here, you might want to put it after that instead of before.